### DISCUSSION TITLE
posts/Go/protobuf/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#346] [Announcements]
https://www.liwenzhou.com/posts/Go/protobuf/ 

 李文周的Blog Go语言 二进制协议 RPC rpc 远程过程调用 IDL protoc protobuf

**Comment by chenshuai1993** [comment]
nice

**Comment by zhougongjin555** [comment]
windows安装protobuf可以参考： https://cloud.tencent.com/developer/article/1821148

**Comment by augustyang** [comment]
nice

**Comment by shihuanx** [comment]
nice
